.gitignore

```diff
@@ -98,3 +98,4 @@ public/system/
 
 ## Sitemap
 public/sitemap.xml
+.byebug_history
```

Comment:
@jcoyne I already amended the PR to remove that change because making a global gitignore made more sense, but I can push a change to .gitignore after this PR is merged.